libs/core/src/components/pds-icon/docs/pds-icon.mdx

```diff
@@ -1,67 +1,79 @@
 import { DocArgsTable, DocCanvas } from '@pine-ds/doc-components';
-import {components as pdsDocsJson} from '@pine-ds/icons/dist/docs.json';
+import { components as pdsDocsJson } from '@pine-ds/icons/dist/docs.json';
 
 # Icon
-Displays Icons used with the Pine Design System
+The icon component is used to display scalable, customizable icons that represent various actions, statuses, or features.
 
 ## Guidelines
 
-Icons are an effective way to enforce the purpose of an experience. Icons are broken out into two main
-categories: Actions and Features. In order to ensure that the icons are adding to the clarity of an experience
-and not creating confusion, we adhere to two principles.
+### When to use
+- Use icons to visually represent actions, features, or common concepts to enhance the user experience.
+- Ideal for buttons, navigation elements, status indicators, or any context where small visual cues can add clarity.
 
-<DocCanvas client:only mdxSource={{
-  react: `<PdsIcon name="thumb-up-filled"></PdsIcon>`,
-  webComponent: `<pds-icon name="thumb-up-filled"></pds-icon>`
-}}>
-  <>
-    <pds-icon name="thumb-up-filled"></pds-icon>
-  </>
-</DocCanvas>
+### When not to use
+- Avoid using icons as the sole means of communication without accompanying text or labels, as this may confuse users.
+- Do not use icons for complex ideas that require explanation or context that cannot be clearly conveyed through visuals alone.
+
+### Accessibility
+- Ensure sufficient contrast between the icon and the background for visibility.
+- Don't rely solely on icons for critical actions.
 
 ## Properties
 
-<DocArgsTable componentName='pds-icon' docSource={pdsDocsJson} />
+<DocArgsTable componentName="pds-icon" docSource={pdsDocsJson} />
 
-### color
-The `color` property allows the icon to render in the specified color. This can be either string or hex format.
+### Color
+
+The `color` property allows the icon to be displayed in a specified color. Accepted formats include:
+
+- String: `red`
+- HEX:  `#FF0000`
+- RGB:  `rgb(255, 0, 0)`
+- HSL: `hsl(0, 100%, 50%)`
+- Pine CSS variables: `var(--pine-color-red-500)`
```

Comment:
I confirmed this works. Should we include these in the examples below?

Reply:
Good call. Added each to the examples.